testnets/nilliontestnet/chain.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "nilliontestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Nillion Testnet",
+  "chain_id": "nillion-chain-testnet-1",
+  "bech32_prefix": "nillion",
```

Comment:
Will be required for registering mainnet here,
please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
PR added https://github.com/satoshilabs/slips/pull/1737

Reply:
looks like you might need to omit the testnet prefix from that repo (but you can keep it here). They won't allow using the same prefix for mainnet and testnet

Reply:
Thanks for the heads up - will remove it from the PR